editoast/editoast_schemas/src/paced_train.rs

```diff
@@ -24,4 +48,92 @@ pub struct PacedTrain {
     pub train_schedule_base: TrainSchedule,
     #[schema(inline)]
     pub paced: Paced,
+    #[serde(default = "Vec::new")]
+    pub exceptions: Vec<PacedTrainException>,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct PacedTrainException {
+    pub id: String,
+    #[schema(inline)]
+    #[serde(flatten)]
+    pub exception_type: PacedTrainExceptionType,
+    pub train_name: Option<TrainNameSilo>,
+    pub rolling_stock: Option<RollingStockSilo>,
+    pub labels: Option<LabelsSilo>,
+    pub category: Option<CategorySilo>,
+    #[schema(inline)]
+    pub speed_limit_tag: Option<SpeedLimitTagSilo>,
+    pub start_time: Option<StartTimeSilo>,
+    pub constraint_distribution: Option<ConstraintDistributionSilo>,
+    pub initial_speed: Option<InitialSpeedSilo>,
+    pub options: Option<OptionsSilo>,
+    pub path_and_schedule: Option<PathAndScheduleSilo>,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq, Eq, ToSchema)]
+#[serde(tag = "type")]
+pub enum PacedTrainExceptionType {
+    #[default]
+    Created,
+    Modified {
+        occurence_index: i32,
+        disabled: bool,
+    },
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, ToSchema)]
+pub struct TrainNameSilo {
+    pub value: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct RollingStockSilo {
+    pub rolling_stock_name: String,
+    pub comfort: Comfort,
+    pub category: Option<RollingStockCategory>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct LabelsSilo {
+    pub value: Vec<String>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct CategorySilo {
+    pub value: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct SpeedLimitTagSilo {
+    #[schema(inline)]
+    pub value: Option<NonBlankString>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct StartTimeSilo {
+    pub value: DateTime<Utc>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct ConstraintDistributionSilo {
+    pub value: Distribution,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct InitialSpeedSilo {
+    pub value: f64,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct OptionsSilo {
+    pub value: TrainScheduleOptions,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, ToSchema)]
+pub struct PathAndScheduleSilo {
```

Comment:
```suggestion
pub struct TripSilo {
```

(not sure about this one)